lightning/send-to-google.js

```diff
@@ -0,0 +1,8 @@
+fn(state => {
+  const { patient, encounter } = state;
+  console.log('Patient', patient);
+  console.log('Encounter', encounter);
+  return state;
+});
+
+// post("blah.com")
```

Comment:
please add a TODO in here to connect to these APIs: https://cloud.google.com/healthcare-api/docs/concepts/fhir#fhir_stores